server/controllers/applications/patchHiredMember.js

```diff
@@ -1,3 +1,40 @@
-module.exports = (req, res) => {
-  res.send('update hire member status');
+const yup = require('yup');
+const { patchHiredMember } = require('../../database/queries/applications/index');
+const updateOfferStatus = require('../../database/queries/offers/updateOfferStatus');
+
+module.exports = (req, res, next) => {
+  const { offer_id: offerId, status } = req.body;
+  const { memberId } = req.params;
+  const schema = yup.object({
+    offerId: yup.number().required(),
+    memberId: yup.string().required(),
+    status: yup.string().required(),
+  });
+  schema
+    .validate({ offerId, memberId, status })
+    .then(() => {
+      patchHiredMember(memberId, offerId, status)
+        .then((result) => {
+          if (result.rowCount && result.rows[0].status === 'accepted') {
+            // update offer status if Hired_member status = accepted
+            updateOfferStatus(offerId, 'completed')
+              .then(() => res.send({
+                error: null,
+                data: result.rows,
+              }))
+              .catch(err => next({ code: 500, msg: err.message }));
```

Comment:
@AymanAlqoqa I think you can remove this line, because an error here would get caught by the handler on line 35.